src/System.Private.ServiceModel/tools/scripts/StartWCFSelfHostedSvcDoWork.cmd

```diff
@@ -4,9 +4,12 @@ setlocal
 REM This script should always be called from StartWCFSelfHostedSvc.cmd
 
 set _setuplog=%~dp0..\..\..\..\SelfHostedWcfServiceSetup.log
+set _setupMonitorFile=%~dp0..\..\..\..\SelfHostedWcfServiceMonitor.log
```

Comment:
For what you are needing to do in the environment we currently have it LGTM, however...

We really should not be chaining scripts together, we have already received feedback from corefx along these lines as well. We should have build.override.targets call each script separately, the scripts can set msbuild properties as needed for additional logic points as we move through the setup and cleanup flow.